### PR TITLE
add graphql steps

### PR DESCRIPTION
**Pull request by ALegchilov**
Before submitting this PR, please ensure that you have completed the following:

- [x] Updated the CHANGELOG.md.
- [x] Checked that there aren't other open pull requests for the same issue/update.
- [x] Checked that your contribution follows the project's contribution guidelines.
- [x] Added corresponding unit/E2E tests
- [x] Unit and E2E pass

## Description

Please describe your changes and any new features you're introducing, or issues you're fixing.

### Related Issues

Please link any related issues or bug reports that this PR will address.

---

Thank you for your contribution!

### CODE REVIEW

File: src/apiConstructionSteps.ts [hunk 1]
```diff
@@ -58,6 +69,28 @@ When('I add body to {string}:', async function (requestKey: string, body: string
...
+ *      characters(page: 2, filter: { name: "rick" }) {
+ *        results {
+ *          name
+ *           }
+ *         }
+ *      }
+ **/
+When('I add {gqlRequestProperty} to GraphQL {string}:', async function (property: string, requestKey: string, valueString: string) {
+  const request: any = await memory.getValue(requestKey);
+  request[property] = (await memory.getValue(valueString));
```
Comment (KirillBogdanets) [inline src/apiConstructionSteps.ts:91]: why do you need brackets here?

File: package.json [hunk 2]
```diff
@@ -11,11 +11,6 @@
     "test:e2e": "qavajs run --config test-e2e/api.ts",
     "debug:e2e": "qavajs run --config test-e2e/api.ts --tags @debug"
   },
-  "husky": {
```
Comment (KirillBogdanets) [inline package.json:14]: why did you remove it?

**Review by AlexGalichenko — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (ALegchilov) [reply package.json:14]: It does not work as is. I would make it work in combination with lint-staged, but @AlexGalichenko decided to get rid of hooks.

Comment (KirillBogdanets) [reply package.json:14]: got it

File: src/apiConstructionSteps.ts (hunk 1 above)
Comment (ALegchilov) [reply src/apiConstructionSteps.ts]: wiped out

**Review by KirillBogdanets — APPROVED** [review]
